server/src/model.ts

```diff
@@ -448,17 +469,25 @@ export async function moveContent({
   id,
   desiredParentFolderId,
   desiredPosition,
-  ownerId,
+  ownerId: loggedInUserId,
+  inLibrary = false, // Not exposed to API call
```

Comment:
Update docstring to include new argument. It was not clear to me at first what "Not exposed to API call" meant.